app/workers/average_exchange_rates_worker.rb

```diff
@@ -0,0 +1,19 @@
+class AverageExchangeRatesWorker
+  include Sidekiq::Worker
+
+  sidekiq_options retry: 1, retry_in: 1.hour
+
+  def perform
+    ExchangeRates::AverageExchangeRatesService.call(force_run: false, selected_date: Time.zone.today.iso8601)
+
+    notify
+  end
+
+  def notify
+    message = 'Average exchange rates for the current period have been added and are accessible for viewing at /exchange_rates.'
```

Comment:
Use active form, instead of passive, when you can: 
'Average exchange rates for the current period are ready. You can view them at  /exchange_rates.'

See: https://www.grammarly.com/blog/active-vs-passive-voice/